### ISSUE TITLE
flora yielding septims should not require IsHarvestable true

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#356]
septims are always harvestable. [Nexus report](https://forums.nexusmods.com/index.php?showtopic=8799893/#entry103649433)

**Comment by SteveTownsend** [comment]
here is the failure log, when _flora_ are set to _No Action_
```
2022-01-08 13:51:19.908    trace  27448 Process REFR 0x000e4877 with base object Coin Purse/0x000d8e7f
2022-01-08 13:51:19.908    trace  27448  object Coin Purse/0x000d8e7f has-ingredient=true
2022-01-08 13:51:19.908     info  27448 REFR 0x000e4877 Base Septim/0x0000000f Object Type septims
2022-01-08 13:51:19.908    trace  27448 flora Coin Purse/0x000d8e7f has lootable Septim/0x0000000f of type 4
2022-01-08 13:51:19.908    trace  27448 TESFormHelper for Septim/0x0000000f set as septims
2022-01-08 13:51:19.908    trace  27448 TESFormHelper for Septim/0x0000000f set as septims
2022-01-08 13:51:19.908    trace  27448 TESValueForm for Septim/0x0000000f has 1
2022-01-08 13:51:19.908    trace  27448 REFR 0x000e4877 with type septims has unharvestable Base Object Coin Purse/0x000d8e7f
```

**Comment by SteveTownsend** [comment]
Fixed for v4.1.0.0 as below
```
2022-01-08 14:06:59.722    trace  27260 Process REFR 0x000e4877 with base object Coin Purse/0x000d8e7f
2022-01-08 14:06:59.722    trace  27260  object Coin Purse/0x000d8e7f has-ingredient=true
2022-01-08 14:06:59.722     info  27260 REFR 0x000e4877 Base Septim/0x0000000f Object Type septims
2022-01-08 14:06:59.722    trace  27260 flora Coin Purse/0x000d8e7f has lootable Septim/0x0000000f of type 4
2022-01-08 14:06:59.722    trace  27260 TESFormHelper for Septim/0x0000000f set as septims
2022-01-08 14:06:59.722    trace  27260 TESFormHelper for Septim/0x0000000f set as septims
2022-01-08 14:06:59.722    trace  27260 TESValueForm for Septim/0x0000000f has 1
2022-01-08 14:06:59.722    trace  27260 No V/W check for objType septims
2022-01-08 14:06:59.722    trace  27260 SmartHarvest Coin Purse/0x000d8e7f for REFR 0x000e4877, collectible=false, type septims, notify false, pending 0
```
